src/blocks/ImageTiles/ImageTiles.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import Ublock from '@src/components/Ublock/Ublock'
+
+export interface ImageTilesProps {
+    uBlockFull: boolean
+    mainData: ImageTilesMainProps
```

Comment:
Remember to always pass simple props. 

Rename it to: 'data'.

Reply:
Done